### ISSUE TITLE
Drush failing to run all of a sudden

### ISSUE THREAD

**Issue by justindasilva** [drush-ops/drush#5015]
**Describe the bug**

All of a sudden, I'm getting the following Drush error when running `drush rs 9001`:
```
TypeError: Drush\Commands\core\RunserverCommands::parseUri(): Argument #1 ($uri) must be of type string, null given, called in /cms-drupal/vendor/drush/drush/src/Commands/core/RunserverCommands.php on line 96
```

**To Reproduce**
Run `drush rs 9001`
(Running this command has worked for months. Just stopped working today suddenly. No idea why.)

**Expected behavior**
My Drupal site starts up at `localhost:9001` like it always has before.

**Actual behavior**
Getting the error I stated above.

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | ^11
| Drupal version? | 9.32
| PHP version     | 8.x
| OS?             | Mac

**Comment by weitzman** [comment]
Fixed in d1fcb2335b. Release coming in next 24 hours or so. Thanks.

**Comment by justindasilva** [comment]
Thank you for the response.

A side question that perhaps you can help me answer. I currently have the following dependency in my `composer.json`:
```
"drush/drush": "^11"
```

I'm just curious what your recommendation might be to make sure I'm always on a stable release.

**Comment by mvggitacc** [comment]
Thank you, fix worked for me.

**Comment by mvggitacc** [comment]
> Fixed in [d1fcb23](https://github.com/drush-ops/drush/commit/d1fcb2335bc87675c1d1aa306f837d00c979c1d9). Release coming in next 24 hours or so. Thanks.

Thank you. The fix worked for me.

**Comment by mvggitacc** [comment]
> Fixed in [d1fcb23](https://github.com/drush-ops/drush/commit/d1fcb2335bc87675c1d1aa306f837d00c979c1d9). Release coming in next 24 hours or so. Thanks.

Thank you. The fix worked for me.
